src/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -49,9 +49,9 @@
  * Static methods to help in access to local file system.
  */
 public class FileStorageUtils {
```

Comment:
We could use SORT_NAME, SORT_DATE and SORT_SIZE in this switch instead of 0, 1 or 2: https://github.com/owncloud/android/blob/c9dada572adfe7d163626d9efe90cedcfd785017/src/com/owncloud/android/utils/FileStorageUtils.java#L219